src/components/bookstate.jsx

```diff
@@ -0,0 +1,22 @@
+import Book from "./book"
+import AddForm from "./AddForm"
+
+const Bookstate = () => {
+    return <>
+    <h2>This is Home page</h2>
+    <Book Name="Eat that frog" Author="Brian Tracy"
+    Genre="Self Improvment"
+    />
+    <Book Name="Automic habits" Author="James Clear"
+    Genre="Self Improvment"
+    />
+    <Book Name="Literary Translation" Author="Hasan ghazala"
+    Genre="Education"
+    />
```

Comment:
- [ ] As specified in the project requirements [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v3_0.md#:~:text=Contains%20a%20list%20to%20renders%20individual%20books), kindly make sure your `Book` component is rendered as a child of `BookList`
like so:
```jsx
const BookList () => {
return(
arr_of_books.map(item => {
<Book book=item />
})
)
}
```
Then the booklist will be the one rendered in `BookState` component.